headers/cub3d.h

```diff
@@ -131,6 +144,7 @@ typedef struct s_info {
 	t_player	player;
 	t_texture	texture[6];
 	bool		active_map;
+	t_ray		ray[1024];
```

Comment:
We should use defined window width instead of static value
```suggestion
	t_ray		ray[W_WIDTH];
```

Reply:
That doesn't work ;(

Reply:
Really? I tried earlier and it worked fine on my linux. I also checked older branches and it was defined this way so I dont know.. Lets solve it next time. But if we can't use defined value, we will have to use a pointer to the array and malloc it on initialization

Reply:
I tried again and that work